src/mockData.js

```diff
@@ -1,5 +1,19 @@
 const movieData = {
   "movies": [
+    {
+      id: 1,
+      title: "Fake Movie Title",
+      poster_path: "https://image.tmdb.org/t/p/original//7G2VvG1lU8q758uOqU6z2Ds0qpA.jpg",
+      backdrop_path: "https://image.tmdb.org/t/p/original//oazPqs1z78LcIOFslbKtJLGlueo.jpg",
+      release_date: "2019-12-04",
+      overview: "Some overview that is full of buzzwords to attempt to entice you to watch this movie! Explosions! Drama! True love! Robots! A cute dog!",
+      average_rating: 6,
+      genres: ["Drama"],
+      budget:63000000,
+      revenue:100853753,
+      runtime:139,
+      tagline: "It's a movie!"
+    },
```

Comment:
Nice work updating the mock data so that we can display more information to the screen!